ReadME.md

```diff
@@ -0,0 +1,98 @@
+
+## Major objective 
+
+- To let beginners venture into the domain of Web Development
+- To facilitate students and working professionals to create decent Resumes by focusing on the content rather than the styling and other UI features
+
+## Tech Stack 
+
+- **Frontend:** HTML5, CSS3, JavaScript, Bootstrap, ReactJS 
+- **Backend:** NodeJS
+
+## Installation Instructions
+
+
+
+## Contribution Guidelines
+
+[![GitHub issues](https://img.shields.io/github/issues/CatsInTech/Rezume?logo=github)](https://github.com/codeuino/Rezume/issues) ![GitHub pull requests](https://img.shields.io/github/issues-pr-raw/CatsInTech/Rezume?logo=git&logoColor=white) ![GitHub contributors](https://img.shields.io/github/contributors/CatsInTech/Rezume?logo=github)  [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat&logo=git&logoColor=white)](https://github.com/CatsInTech) 
+
+ [![GitHub last commit](https://img.shields.io/github/last-commit/CatsInTech/Rezume?logo=github)](https://github.com/CatsInTech) [![CatsInTech](https://img.shields.io/badge/Author-@praveenscience-gray.svg?colorA=gray&colorB=dodgerblue&logo=github)](https://github.com/CatsInTech/) 
+
+- Please read our [Code of Conduct](https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md) and [Gitflow](https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md) for contributing towards the project.
+- Write clear meaningful git commit messages (Do read [this](http://chris.beams.io/posts/git-commit/)).
+- Make sure your PR's description contains GitHub's special keyword references that automatically close the related issue when the PR is merged. (Check [this](https://github.com/blog/1506-closing-issues-via-pull-requests) for more info)
+- When you make very very minor changes to a PR of yours (like for example fixing a text in button, minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at [here](https://davidwalsh.name/squash-commits-git))
+- When you're submitting a PR for a UI-related issue, it would be really awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker.
+- Please follow the [PR Template](https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md) to create the PR.
+- Always create PR to `develop` branch.
+
+
+> **Note:** Are you an **absolute beginner?** Please [check the issues](https://github.com/CatsInTech/Rezume/issues) labeled with `first-timers-only`. These issues might _let your Open Source journey begin_ - **Welcome onboard!**
```

Comment:
We can start with with `beginners` label itself. And a couple of grammatical fixes below:

```suggestion
> **Note:** Are you an **absolute beginner**? Please [check the issues](https://github.com/CatsInTech/Rezume/issues) labeled with `beginners`. These issues might _let your Open Source journey begin_ - **Welcome on-board!**
```